src/components/carousel/variants/CarouselWrapper.tsx

```diff
@@ -0,0 +1,48 @@
+import Box from '@mui/material/Box';
+
+import Carousel, { useCarousel } from '@src/components/carousel/index';
+import CarouselNavigationArrows from '@src/components/carousel/components/CarouselNavigationArrows.tsx';
+import { CarouselSection } from '@src/components/poster/carousel-section.tsx';
+import { useItemsPerSlide } from '@src/hooks/components/use-item-per-slide';
+import { useChunkedData } from '@src/hooks/components/use-chunked-data';
+
+import { CarouselWrapperProps } from './types';
+
+export default function CarouselWrapper<T>({
+                                             data,
+                                             title,
```

Comment:
to favor readability could be a better approach if we detruct the props inside te function? eg:

{
 ...
} = props